### PR TITLE
Create Routes

### PR DESCRIPTION
**Pull request by TilegenGena**
Summary:
- Create routes * dashboard * login * user-profile
- Add notification icon
- Reorder `app.component.html`
- Move `mentorship.app.module.ts` into `mentorship` folder

Test:
Run Server, Client
Main page with links (dashboard, user-profile, login) leads to new pages

Снимок экрана 2024-01-10 в 22 34 20

### CODE REVIEW

File: client/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+import { DashboardComponent } from './dashboard/dashboard.component';
+import { LoginComponent } from './login/login.component';
+import { UserProfileComponent } from './user-profile/user-profile.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'dashboard', component: DashboardComponent },
+  { path: 'login', component: LoginComponent },
+  { path: 'user-profile', component: UserProfileComponent },
+];
```
Comment (hopeaisulu) [inline client/src/app/app-routing.module.ts:9]: Eventually there will be only two pages, Dashboard and Advice wall. Other will be a button with a modal pop up window. You can remove login and user profile from the route. But make a buttons: Mentors, Mentorship, Bell icon, button with the username, logout button.

So combining these pages and button will look as follows:
Dashboard, Advice Wall, Mentors, Mentorship, bell icon, Aisulu Abdraimova, logout button.

File: client/src/app/app.component.html [hunk 2]
```diff
@@ -1,2 +1,7 @@
-<router-outlet></router-outlet>
-<app-mentorship-app-root></app-mentorship-app-root>
\ No newline at end of file
+<app-mentorship-app-root></app-mentorship-app-root>
+<nav>
+  <a routerLink="/dashboard"> Dashboard</a>
+  <a routerLink="/login"> Login</a>
+  <a routerLink="/user-profile"> User Profile</a>
+</nav>
```
Comment (hopeaisulu) [inline client/src/app/app.component.html:5]: please remove extra pages and add buttons.You can rename one of these to "Advice Wall" or create new.

File: client/src/app/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>login works!</p>
```
Comment (hopeaisulu) [inline client/src/app/login/login.component.html]: For now we don't need this login page, please remove, we will add later.

**Review by hopeaisulu — CHANGES_REQUESTED** [review]
Looks great!
Please remove for now the extra components that we don't need.

**Review by hopeaisulu — APPROVED** [review]

**Comment by networkflow** [comment]
Thanks for including a screenshot on the PR, very helpful! Overall looks pretty good to me, it's still early in the app so I'm not 100% sure what your plans are but seems like a good start.

**Pull request merged**